README.md

```diff
@@ -151,6 +151,51 @@ and run:
 _build/bun install
 ```
 
+### Replace esbuild with bun
+
+You can use `bun` to build CSS with TailwindCSS, replacing both `esbuild` and the `tailwindcss` library in Elixir.
+
+First, update `assets/package.json`:
+
+```json
+"dependencies": {
+  "phoenix": "workspace:*",
+  "phoenix_html": "workspace:*",
+  "phoenix_live_view": "workspace:*",
+  "tailwindcss": "^3.4.9",
+  "topbar": "^3.0.0"
+}
+```
```

Comment:
IIRC by default there is no `package.json` currently, so it probably should be full file or should add extra steps to have this file in place. Especially that this requires extra configuration for `workspace:*` to work.

Reply:
I was thinking of this subsection as a continuation of the one before where the package.json is introduced.

Also, if one does not add the package.json in the step before, how would they use bun with phoenix?

Reply:
Ahhh, I missed that it is described above.